examples/asr/speech_multitask/speech_to_text_aed.py

```diff
@@ -66,6 +67,14 @@ def main(cfg):
 
     trainer = pl.Trainer(**cfg.trainer)
     exp_manager(trainer, cfg.get("exp_manager", None))
+
+    # Check for canary spl tokens to create spl_tokenizer.
+    if cfg.get("spl_tokens"):
+        spl_cfg = cfg["spl_tokens"]
+        spl_path, task_tokens, lang_tokens = spl_cfg.get('dir'), spl_cfg.get("task_tokens"), spl_cfg.get("lang_tokens")
+        CanaryTokenizer.build_special_tokenizer(tokens=task_tokens + lang_tokens, output_dir=spl_path)
```

Comment:
@pzelasko - this isn't correct example scripts cannot specialize to Canary. Import the class dynamically from config, assume there's a base class to our Canary tokenizer that has this build_special_tokenizer() as an abstract function, and fall it here in generic way

Reply:
Would this work if we just change CanaryTokenizer to a more generic name?